src/pkg/packager/common.go

```diff
@@ -474,7 +474,13 @@ func (p *Packager) validatePackageChecksums() error {
 		itemPath := strs[1]
 		expectedShasum := strs[0]
 
-		actualShasum, err := utils.GetSHA256OfFile(filepath.Join(p.tmp.Base, itemPath))
+		path := filepath.Join(p.tmp.Base, itemPath)
+		if utils.InvalidPath(path) {
+			message.Debugf("skipping checksum of %s as it does not exist", path)
+			continue
+		}
```

Comment:
Thinking about this some more I think it would be better to have some logic that checks / skips the components /images that it doesn't expect to be there locally - we already have the YAML at this point in loading so if we passed the component list to this function (or just accessed the options) we could figure this out.

That would likely make sense in a helper function too since very similar logic would be needed / exists in the `getLayersToPullFromRequestedComponents` function (and it would help make that one smaller too).